src/System.Private.ServiceModel/tests/Common/src/BridgeClient.cs

```diff
@@ -58,7 +65,18 @@ private static void MakeConfigRequest()
                 {
                     var response = httpClient.PostAsync("/config/", content).Result;
                     if (!response.IsSuccessStatusCode)
-                        throw new Exception("Unexpected status code: " + response.StatusCode);
+                    {
+                        string reason = String.Empty;
+                        if (response.Content != null)
+                        {
+                            string contentAsString = response.Content.ReadAsStringAsync().Result;
+                            reason = String.Format("{0}Bridge returned content:{0}{1}",
+                                                    Environment.NewLine, contentAsString);
```

Comment:
You've dropped the status code. We should have the status code and the status reason included.